### DISCUSSION TITLE
Regarding Pyautogen Studio's "State Transition" option

### DISCUSSION THREAD

**Discussion by AsswinCR** [microsoft/autogen#2665] [General]
I am currently working with  Pyautogen Studio and I was wondering if the State Transition option is available in the in it. Can you please confirm if this feature is included in Pyautogen Studio?. As i can only see 3 options which are 1) Auto 2) Round Robin and 3) Random. It would be nice if they include state transition option too.

**Comment by ekzhu** [comment]
It's not currently available
